Cargo.toml

```diff
@@ -7,25 +7,29 @@ description = "Tool to find and fix Rust formatting issues"
 repository = "https://github.com/rust-lang/rustfmt"
 readme = "README.md"
 license = "Apache-2.0/MIT"
-build = "build.rs"
+build = "rustfmt-core/rustfmt-bin/build.rs"
 categories = ["development-tools"]
 edition = "2018"
 
 [[bin]]
 name = "rustfmt"
-path = "src/bin/main.rs"
+path = "rustfmt-core/rustfmt-bin/src/bin/main.rs"
+test = false
```

Comment:
Disabled all the tests in the root package as they will be tested in rustfmt-core.